### PR TITLE
Replace the gremlin-server.sh with a janusgraph-server.sh

### LINKED ISSUE #2119: Create a central entrypoint a JanusGraph server
**Describe the feature:**
Replace our default gremlin server startup with janusgraph management part.

**Describe a specific use case for the feature:**
* Add new GraphManager goal is to replace the cfg JanusGraphManager and the default tinkerpop with single and use this for any case.
* Add a new entry to start JanusGraph and ensure default configurations.

### PR DESCRIPTION
**Pull request by farodin91**
Partial fixes #2119

* This adds a replacement for the current gremlin server with a improve configurations possibilities for java, such as, gc config (here copied from cassandra)
* Add tests for janusgraph-server.sh

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?
- [x] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by farodin91** [comment]
I will rebase this pr after #2135 is merged.

File: docs/basics/server.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ curl -XPOST -Hcontent-type:application/json -d *{"gremlin":"g.V().count()"}* [IP
 
 ## JanusGraph Server as Both a WebSocket and HTTP Endpoint
 
-As of JanusGraph 0.2.0, you can configure your `gremlin-server.yaml` to
+As of JanusGraph 0.2.0, you can configure your `janusgraph-server.yaml` to
```
Comment (porunov) [inline docs/basics/server.md:293]: As I understand as of 0.2.0 it is still `gremlin-server.yaml` but as for `0.6.0` it is `janusgraph-server.yaml`

File: docs/basics/server.md [hunk 2]
```diff
@@ -514,7 +514,7 @@ available to scripts provided to Gremlin Server - `graph` and `g`.
 At this point, Gremlin Server is configured and can be used to connect
 to a new or existing JanusGraph database. To start the server:
 ```bash
-$ bin/gremlin-server.sh conf/gremlin-server-janusgraph.yaml
+$ bin/gremlin-server.sh console conf/gremlin-server-janusgraph..yaml
```
Comment (porunov) [inline docs/basics/server.md:517]: Remove additional dot. I.e. from `gremlin-server-janusgraph..yaml` to `gremlin-server-janusgraph.yaml`.
Also, `gremlin-server.sh` doesn't accept `console` parameter. I guess we need to change it to `janusgraph-server.sh`

Comment (farodin91) [reply docs/basics/server.md]: I revert this change. I already created a ticket for this section, see issue #2146.

File: docs/basics/server.md (hunk 1 above)
Comment (farodin91) [reply docs/basics/server.md]: I revert this change.

**Comment by farodin91** [comment]
I added a copyright mark to the janusgraph-server.sh.

**Comment by farodin91** [comment]
@porunov Do you like to review this PR? We have the new startup script in our production environment for a month.

**Review by porunov — APPROVED** [review]
LGTM. Thank you @farodin91 !

File: docs/changelog.md [hunk 3]
```diff
@@ -73,6 +73,15 @@ For more information on features and bug fixes in 0.6.0, see the GitHub mileston
 
 #### Upgrade Instructions
 
+##### Replacing the server startup script is replaced
+
+The `gremlin-server.sh` is placed by `janusgraph-server.sh`. The
```
Comment (li-boxuan) [inline docs/changelog.md:78]: Typo? "placed" -> "replaced".

**Review by li-boxuan — COMMENTED** [review]
@farodin91 I am commenting on this old thread, mainly because I found that we replaced "gremlin-server.sh" with "janusgraph-server.sh", but retained "gremlin-server.bat". This might confuse windows users... is the bat file still usable? Is it possible to update the bat file as well?

**Pull request merged**